WorkloadManagementSystem/JobWrapper/JobWrapper.py

```diff
@@ -429,7 +431,7 @@ def execute(self):
         outputs = threadResult['Value']
     else:  # if the execution thread didn't complete
       self.log.error('Application thread did not complete')
-      self.__report('Failed', 'Application thread did not complete', sendFlag=True)
+      self.__report(status='Failed', minorStatus=JobMinorStatus.APP_THREAD_NOT_COMPLETE, sendFlag=True)
       self.__setJobParam('ApplicationError', 'Application thread did not complete', sendFlag=True)
```

Comment:
```suggestion
      self.__setJobParam('ApplicationError', JobMinorStatus.APP_THREAD_NOT_COMPLETE, sendFlag=True)
```

Reply:
I didn't dare using a job minor status as a job parameter! but you are right, the string being the same, why not!

Reply:
Done